checks.d/mysql.py

```diff
@@ -410,7 +410,7 @@ def _collect_metrics(self, host, db, tags, options, queries):
         results = self._get_stats_from_status(db)
         results.update(self._get_stats_from_variables(db))
 
-        if self._is_innodb_engine_enabled(db):
+        if (not _is_affirmative(options.get('disable_innodb_metrics', False)) and self._is_innodb_engine_enabled(db)):
```

Comment:
let's document the option in the yaml example: https://github.com/DataDog/dd-agent/blob/jaime/innodbfix/conf.d/mysql.yaml.example#L19